Eastern-Research-Group/csb-rebate-forms-app
# Feature/update nces json

## Related Issues:
* CSBAPP-229

## Main Changes:
Add new A999999 NCES ID record to the nces.json file (update to #347)

## Steps To Test:
1. Navigate to a 2023 submission within the app. Enter in "A999999" as the NCES ID. Once the 2023 FRF definition has been updated to accommodate it, the NCES fields should respond appropriately (be editable and/or any other logic OAR has conveyed).

## TODO:
- [ ] Ensure the source Excel/CSV file is updated with the new A99999 row at the top, and ensure all columns/fields are filled in appropriately (which may mean re-generating the JSON file if certain field values should change)
